Assets/Shopify/UIToolkit/Shops/Generic/Classes/MultiProductListItem.cs

```diff
@@ -10,15 +10,17 @@ namespace Shopify.UIToolkit.Shops {
 
     public class MultiProductListItem : MonoBehaviour {
         public RemoteImageLoader ImageLoader;
+        public UnityEngine.UI.Image Image;
+        public Sprite ErrorImage;
         public Text TitleLabel;
         public Text PriceLabel;
         public Text DescriptionLabel;
-
+        public AspectRatioFitter AspectRatioFitter;
         public UnityAction OnClick;
-
         private const int MaxDescriptionCharacters = 80;
 
         public void SetProduct(Product product) {
+            Image.CrossFadeAlpha(0f, 0.01f, true);
```

Comment:
you'd think i could do `image.color = new Color(1,1,1,0);` here but the truth is that `CrossFadeAlpha` sets the color of the renderer instead of the graphic so this is the easiest way to set it without needing to grab a ref to the renderer component.

Reply:
Also we do this in set product so that when a new product is assigned we arent showing the old product image while this one loads